src/devices/Common/Iot/Device/Common/NumberHelper.cs

```diff
@@ -9,7 +9,7 @@ namespace Iot.Device.Common
     /// <summary>
     /// Helpers for number.
     /// </summary>
-    internal static class NumberHelper
+    public class NumberHelper
```

Comment:
Why did we remove static? Why did we change to public?

Reply:
I had problems with the `internal` and just copied and pasted what I had in `WeatherHelper`. No problems, I'll change it to static too.

Reply:
Done (pending commit)